app/services/MultiTenantService.rb

```diff
@@ -0,0 +1,15 @@
+class MultiTenantService
```

Comment:
Can we change the name of the file to be snakecase to match Rail's convention. `multi_tenant_service.rb` resolves to this same class name

Reply:
Since this is related to organizations, or tenants maybe you can namespace this to start building a hierarchal structure like:
```
Organizations::EmailService
Tenants::EmailService
```

On another note, I think naming this to be specific to emails might make sense. Unless the intention is to extend this to address things beyond emails eventually?

Reply:
Yeah, I was thinking it might be nice to centralize it so I kept it generic for now